fineract-provider/src/main/java/org/apache/fineract/infrastructure/event/external/service/ExternalEventsConfigurationMapper.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.fineract.infrastructure.event.external.service;
+
+import java.util.List;
+import org.apache.fineract.infrastructure.event.external.data.ExternalEventConfigurationItemData;
+import org.apache.fineract.infrastructure.event.external.repository.domain.ExternalEventConfiguration;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface ExternalEventsConfigurationMapper {
```

Comment:
Can you please `MappingConstants.ComponentModel.SPRING` instead of the magic "spring" literal?
Also, use `unmappedTargetPolicy = ReportingPolicy.ERROR` so the build fails if something is not mapped properly.

Btw you could also create a very generic AbstractMapper in Fineract that holds these configurations. An example what I did is for the Avro mappings (AvroMapperConfig.java and for example look at ClientDataMapper.java how to use it)